BUILD

```diff
@@ -107,23 +107,30 @@ config_setting(
     values = {"compilation_mode": "fastbuild"},
 )
 
+# Synthesize a copy of the file in the current package so it can be embedded.
+genrule(
+    name = "aws_rds_certs",
+    srcs = ["@aws_rds_certs//file:rds-combined-ca-bundle.pem"],
+    outs = ["rds-combined-ca-bundle.pem"],
+    cmd_bash = "cp $(SRCS) $@",
+)
+
 # N.B. this is ignored by gazelle so must be updated by hand.
 # It must live at the repo root to be able to bundle other files using
 # "go:embed".
 go_library(
     name = "bundle",
     srcs = ["bundle.go"],
-    embedsrcs = select({
-        ":fastbuild": [
-            "//:config_files",
-            "//static",
-        ],
+    embedsrcs = [
+        "//:config_files",
+        "//static",
+        ":rds-combined-ca-bundle.pem",
```

Comment:
can we put this in //static? I'd like to avoid having every embedded file need to be separately listed here (if possible)

Reply:
It feels a bit odd putting this under static. 
Static files are those that are meant to be served by the UI so it feels wrong to lump in the certs under there.

What's the concern about adding individual files into the bundle?

Reply:
Would you be open to putting this in config_files if you feel that //static is a bad path? Or adding a filegroup like "//certs" instead?

Because we already do some switching on buildmode to decide what to include, my concern is I don't want people to have to think about build_modes x number_of_files. It's easier if you can just be like... oh yeah we need certs, configs, and static files.

Reply:
I moved the cert into //:embedded_certs file group instead of depending on the file directly.